config/service.go

```diff
@@ -55,6 +55,30 @@ type ServiceTelemetryLogs struct {
 	// Encoding sets the logger's encoding.
 	// Valid values are "json" and "console".
 	Encoding string `mapstructure:"encoding"`
+
+	// DisableCaller stops annotating logs with the calling function's file
+	// name and line number. By default, all logs are annotated.
+	DisableCaller bool `mapstructure:"disableCaller"`
```

Comment:
I like the comment of explaining the default behavior, but as engineer I would like to see the default "value" as well. Something like:

```
(default = true)
```

Or however you want.

Reply:
Sure, added it.